src/components/StatueDetailContainer.js

```diff
@@ -1,14 +1,21 @@
 import React, { Component } from "react";
-import style from "./StatueDetailContainer.module.css";
+import "./StatueDetailContainer.module.css";
 
 export class StatueDetailContainer extends Component {
   render() {
     const { name, details } = this.props.statue;
 
     return (
-      <div className={style.container}>
-        <div>{name}</div>
-        <div>{details}</div>
+      <div className="col-lg-8 col-md-6 col-sm-12">
```

Comment:
Perhaps go for `col-lg-9` as the corresponding column is `col-lg-3`, so there is no column gap (if we needed a gap better to use padding/margin).
Also perhaps go for `col-md-12` as the half and half at the medium breakpoint seems really tiny